AutomatedTesting/Gem/PythonTests/Prefab/tests/overrides/EditEntity_UnderNestedInstance.py

```diff
@@ -0,0 +1,102 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+def EditEntity_UnderNestedInstance():
+    """
+    Test description:
+    - Creates a "NestedPrefabs_Prefab_3" prefab containing one entity called "Test_Entity".
+    - Creates a "NestedPrefabs_Prefab_2" prefab containing "NestedPrefabs_Prefab_3" instance.
+    - Creates a "NestedPrefabs_Prefab_1" prefab containing "NestedPrefabs_Prefab_2" instance.
+    - Instantiates the "NestedPrefabs_Prefab_1" prefab twice under the level prefab.
+    - Edits the "Test_Entity" of the first "NestedPrefabs_Prefab_3" instance.
+    - Checks that after propagation, the "Test_Entity" of the first "NestedPrefabs_Prefab_3" instance contains the correct edits.
+    - Checks that after propagation, the "Test_Entity" of the second "NestedPrefabs_Prefab_3" instance does not contain any edits.
+
+    - Level (Focused prefab)
```

Comment:
This test doesn't add enough variation imo. Rather than always making the level be the focused prefab, this test can make 'NestedPrefabs_Prefab_1' be the focused prefab.

That will validate that overrides work on any prefabs and not just the level.

There should be a script exposed function to can call to set the focused prefab. It's already used in other tests.

Reply:
If just the focused prefab is changed to 'NestedPrefabs_Prefab_1', the actual template will propagate to both instances and the leaf entity will change for all instances. But I will modify this test some more to work with the focus being on an instance other than the level.